spec/index.html

```diff
@@ -193,7 +195,13 @@ <h2>Notation and Terminology</h2>
       <dfn data-cite="RDF12-CONCEPTS#dfn-generalized-rdf-graph">generalized RDF graphs</dfn>, and
       <dfn data-cite="RDF12-CONCEPTS#dfn-generalized-rdf-dataset">generalized RDF datasets</dfn>.</p>
 
-    <p>An <dfn class="export">interpretation</dfn> is a mapping from IRIs and literals into a set,
+    <p>A <dfn>ground RDF graph</dfn> is an RDF graph that no blank nodes appear in.
+      A <dfn>ground RDF term</dfn> is an RDF term that no blank nodes appear in.
+      A <dfn>ground triple term</dfn> is a triple term  term that no blank nodes appear in.
+      A <dfn>ground RDF triple</dfn> is an RDF triple that no blank nodes appear in.</p>
```

Comment:
These might be better as items in a list, rather than a streaming paragraph.
```suggestion
    <p>A <dfn>ground RDF graph</dfn> is an RDF graph in which no blank nodes appear.
      A <dfn>ground RDF term</dfn> is an RDF term in which no blank nodes appear.
      A <dfn>ground triple term</dfn> is a triple term term in which no blank nodes appear.
      A <dfn>ground RDF triple</dfn> is an RDF triple in which no blank nodes appear.</p>
```